frontend/ui/elements/common_info_menu_table.lua

```diff
@@ -49,5 +51,32 @@ common_info.report_bug = {
         })
     end
 }
+if Device:isKobo() then
+    common_info.reboot = {
+        text = _("Reboot"),
+    	  callback = function()
+            Screen:setRotationMode(0)
+            require("ui/screensaver"):show("reboot", _("Rebooting..."))
+            Screen:refreshFull()
+            UIManager:nextTick(function()
+                UIManager:broadcastEvent(Event:new("Close"))
+                Device:reboot()
+            end)
+        end
+    }
+
+    common_info.poweroff = {
+        text = _("Power off"),
+    	  callback = function()
+            Screen:setRotationMode(0)
+            require("ui/screensaver"):show("poweroff", _("Powered off"))
+            Screen:refreshFull()
+            UIManager:nextTick(function()
+                UIManager:broadcastEvent(Event:new("Close"))
+                Device:powerOff()
+            end)
```

Comment:
Can we reuse the function `poweroff_action` from UIManager instead? This, it's easier for us to push bug fix or update to this piece of code.